content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -20,33 +20,37 @@ A [`rippled` server](xrpl-servers.html) running in [validator mode](rippled-serv
 
 Even if your validator isn't a _trusted_ validator, it stills plays an important role in the overall health of the network. These validators help set the standard that trusted validators are measured against. For example, if a trusted validator is disagreeing with a lot of these validators that aren't listed in UNLs, that might indicate a problem.
 
+**Warning:** Validators should not be accessible to the public. Do not allow public WebSocket access to your validator server or any other form of public access.
+
 
 
 ## 1. Understand the traits of a good validator
 
-Strive to have your validator always embody the following properties. Being a good validator helps `rippled` server operators and validator list publishers, such as [https://vl.ripple.com](https://vl.ripple.com) and [https://vl.xrplf.org](https://vl.xrplf.org), to trust your validator before adding it to their UNLs.
+Strive to have your validator always embody the following properties. Being a good validator helps `rippled` server operators and validator list publishers (such as https://vl.coil.com, https://vl.ripple.com, and https://vl.xrplf.org), to trust your validator before adding it to their UNLs.
 
 - **Available**
 
     A good validator is always running and submitting validation votes for every proposed ledger. Strive for 100% uptime.
 
 - **In agreement**
 
-    A good validator's votes match the outcome of the consensus process as often as possible. To do otherwise could indicate that your validator's software is outdated, buggy, or intentionally biased. Always run the [latest `rippled` release](https://github.com/ripple/rippled/tree/master) without modifications. [Watch `rippled` releases](https://github.com/ripple/rippled/releases) to be notified of new releases.
+    A good validator's votes match the outcome of the consensus process as often as possible. To do otherwise could indicate that your validator's software is outdated, buggy, or intentionally biased. Always run the [latest `rippled` release](https://github.com/XRPLF/rippled/tree/release) without modifications. [Watch `rippled` releases](https://github.com/XRPLF/rippled/releases) and subscribe to the [Google Group](https://groups.google.com/g/ripple-server) to be notified of new releases.
 
 - **Issuing timely votes**
 
     A good validator's votes arrive quickly and not after a consensus round has already finished. To keep your votes timely, make sure your validator meets the recommended [system requirements](system-requirements.html), which include a fast internet connection.
 
+Due to the need for consistent performance, it is not recommended to submit new transactions to, or query data from, a validator. A validator should be dedicated to participating in consensus.
```

Comment:
This is debatable. The point of the "wallet server" configuration was to lower the barriers for businesses running validators, since having an entire separate machine for it is a big expense and if you're only doing light API queries (submitting occasional transactions and checking the balance and transactions of your own account) it's reasonable to expect "good enough" validation performance from a server in that configuration. (If it's powerful enough to handle your queries, it's probably powerful enough to do validation too.)

In an ideal world every validator would be perfect, but in the real world I think it's a fair and pragmatic approach for the network to have a larger population of validators run by different entities that have an interest in the XRP Ledger. Rather than putting validators on a pedestal, let's say these are goals and take a softer stance on servers that do both. Also, indent 4 spaces so it's a continuation of the previous bullet point.
 
```suggestion
    It is possible to submit new transactions to, query data from, a validator, but heavy loads of API queries may make the validator less reliable at keeping up with consensus. If your API needs are light enough you can use a server for both purposes, but ideally a validator should be dedicated to participating in consensus.
```